modules/exploits/windows/local/cve_2020_1170_windows_defender_lpe.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::EXE # Needed for generate_payload_dll
+  include Msf::Post::Windows::FileSystem
+  include Msf::Post::Windows::FileInfo # Needed to retrieve info on the file version for MpCmdRun.exe
+  include Msf::Post::Windows::ReflectiveDLLInjection
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(
+        update_info(
+          info,
+          'Name' => 'Windows Defender MpCmdRun.log Arbitrary Folder Deletion Vulnerability',
```

Comment:
Suggest:

```suggestion
          'Name' => 'Windows Defender MpCmdRun.log Arbitrary Folder Deletion Privilege Escalation',
```

This `exploit` module is obviously an exploit. A vulnerability is implied. The module file name also uses `lpe` terminology.

Reply:
I think I left it that way as this is still a WIP module and its possible that we could have to change the title if the LPE doesn't work for some reason. So that is why I left it as that for the time being. I'll change it over though.